external/twitter/src/main/scala/org/apache/spark/streaming/twitter/TwitterInputDStream.scala

```diff
@@ -28,11 +28,16 @@ import org.apache.spark.storage.StorageLevel
 import org.apache.spark.Logging
 import org.apache.spark.streaming.receiver.Receiver
 
-/* A stream of Twitter statuses, potentially filtered by one or more keywords.
+/** Bounding box to filter tweets by location. Units are degrees. */
+case class BoundingBox(west: Double, south: Double, east: Double, north: Double)
```

Comment:
What if I want to filter using a non-contiguous surface (e.g. I want to know who talks about soccer in Manchester or Chelsea during a football match betw. their 2 teams) ?